doc/tutorials/enable_s5.rst

```diff
@@ -0,0 +1,364 @@
+to User VM through channel, if User VM receives the command, it will send an "ACK"
+to Device Model. It is the responsibility of Service VM to check if the User VMs
+shutdown successfully or not, and decides when to power off itself.
+
+What does "life cycle manager" in User VM do?
+=============================================
+
+As current S5 reference design, there is a life-cycle-manager daemon running in
+User VM to implement S5. It will wait for the command from Service VM on the
+paired serial port, there is a simple protocol between Service VM and User VM:
+when it receives ``shutdown``, then send "acked" to Service VM, then it can power off
+User VM.
+
+However, if the User VM is not ready to power off, it can ignore the ``shutdown`` command.
+
+.. _enable_s5:
+
+Enable S5
+*********
+
+Enable S5 based on the the particular OS:
+
+* For Linux (LaaG) or Windows (WaaG), refer to the following configurations in
+  ``devicemodel/samples/nuc/launch_uos.sh`` launch script for ``acrn-dm``.
+
+  .. code-block:: none
+     :emphasize-lines: 2,3,4,17
+     :name: laag-waag-script
+     :caption: LaaG/WaaG launch script
+  
+     #for pm by vuart setting
+     pm_channel="--pm_notify_channel uart "
+     pm_by_vuart="--pm_by_vuart pty,/run/acrn/life_mngr_"$vm_name
+     pm_vuart_node=" -s 1:0,lpc -l com2,/run/acrn/life_mngr_"$vm_name
+     
+     #for memsize setting
+     mem_size=2048M
+     
+     acrn-dm -A -m $mem_size -s 0:0,hostbridge \
+       -s 2,pci-gvt -G "$2" \
+       -s 5,virtio-console,@stdio:stdio_port \
+       -s 6,virtio-hyper_dmabuf \
+       -s 3,virtio-blk,/home/clear/uos/uos.img \
+       -s 4,virtio-net,tap0 \
+       -s 7,virtio-rnd \
+       --ovmf /usr/share/acrn/bios/OVMF.fd \
+       $pm_channel $pm_by_vuart $pm_vuart_node \
+       $logger_setting \
+       --mac_seed $mac_seed \
+       $vm_name
+     }
+
+* For RT-Linux, refer to the ``devicemodel/samples/nuc/launch_hard_rt_vm.sh`` script:
+
+  .. code-block:: none
+     :emphasize-lines: 2,3,12
+     :name: rt-script
+     :caption: RT-Linux launch script
+  
+     # for pm setting
+     pm_channel="--pm_notify_channel uart "
+     pm_by_vuart="--pm_by_vuart tty,/dev/ttyS1"
+     
+     /usr/bin/acrn-dm -A -m $mem_size -s 0:0,hostbridge \
+        --lapic_pt \
+        --rtvm \
+        --virtio_poll 1000000 \
+        -U 495ae2e5-2603-4d64-af76-d4bc5a8ec0e5 \
+        -s 2,passthru,02/0/0 \
+        -s 3,virtio-console,@stdio:stdio_port \
+        $pm_channel $pm_by_vuart \
+        --ovmf /usr/share/acrn/bios/OVMF.fd \
+        hard_rtvm
+     
+     }
+
+  .. note:: For RT-Linux, the vUART is emulated in hypervisor, expose the node as ``/dev/ttySn``.
+
+#. For LaaG and RT-Linux VM, a reference daemon is offered to run:
+
+   a. Use below command lines to make ``life_mngr``.
+
+      .. code-block:: none
+   
+         $ cd acrn-hypervisor/misc/life_mngr
+         $ gcc life_mngr.c -o life_mngr
```

Comment:
@mcao6 , can we just use the following command lines instead ?
`cd acrn-hypervisor/misc/life_mngr`
`make life_mngr`

Reply:
just checked, you can just use: 
   make

Reply:
@mgcao, can we use `make life_mngr` instead ?  because I saw the error mesage when I just use `make` in this directory: `acrn-hypervisor/misc/life_mngr` 
```
x86_64-w64-mingw32-gcc -g life_mngr_win.c -o /root/workspace/acrn-hypervisor/misc/life_mngr/build/life_mngr_win.exe -Wall -O2 
life_mngr_win.c:6:10: fatal error: windows.h: No such file or directory
    6 | #include <windows.h>
      |          ^~~~~~~~~~~
compilation terminated.
make: [Makefile:42: all] Error 1 (ignored)
```

Reply:
if you want to build windows version, you need follow the step in this guide to do that.

Reply:
Yes, there is a note to fix this issue in the next paragraph.